test/inclusivity.jl

```diff
@@ -1,3 +1,5 @@
+Inclusivity(args...) = @test_deprecated Intervals.Inclusivity(args...)
```

Comment:
Why define this function, doesn't seem to be used?

Reply:
There are many calls to `Inclusivity` in this file. I did this to avoid having to do `(@test_deprecated Inclusivity(...))` for each one